sdk/web-pubsub/web-pubsub/review/web-pubsub.api.md

```diff
@@ -25,6 +25,7 @@ export interface GenerateClientTokenOptions extends OperationOptions {
     groups?: string[];
     roles?: string[];
     userId?: string;
+    webPubSubClientAccess?: WebPubSubClientAccess;
```

Comment:
the name here is a little repetitive and not very descriptive. How about `protocol?: WebPubSubClientProtocol` ? would that work? Looks like `mqtt` is a messaging protocol

Reply:
@maorleger yeah that makes sense. I'll sync with the other SDK teams about this to make sure the naming is consistent across the repos

Reply:
@maorleger I've updated the name. Can you please check again? Thanks